### DISCUSSION TITLE
AptX A2DP sink are able in any example?

### DISCUSSION THREAD

**Discussion by josjkasan** [pschatzmann/arduino-audio-tools#699] [Q&A] [answered]
Hello! I just try example of A2DP I2S example, but AptX codecs not able. What should I use to get AptX codecs work? Or this library doesn`t have this function?

**Comment by pschatzmann** [comment, marked as answer]
No, the ESP32 A2DP functionality does not provide any other codecs then SBC

**Reply by josjkasan** [reply]
But, there was posted library arduino-libopenaptx, and says that lib implemented in this lib, but I could not figure how to use AptX

**Reply by pschatzmann** [reply]
that is just a codec! The A2DP stack is closed source and if Espressif does not provide this functionality there is nothing you can do!

**Reply by josjkasan** [reply]
Thanks, that is why I can`t figure it out!
